config/locales/es.yml

```diff
@@ -1002,8 +1002,8 @@ es:
       total_items: ARTÍCULOS TOTALES
       supplier_totals: Totales de proveedores del Ciclo de Pedido
       supplier_totals_by_distributor: Totales de proveedores por Distribuidor del Ciclo de pedidos
-      totals_by_supplier: Totales del distribuidor por proveedor del Ciclos de Pedido 
-      customer_totals: Totales de los consumidores del Ciclo de Pedido 
+      totals_by_supplier: Totales del distribuidor por proveedor del Ciclos de Pedido
+      customer_totals: Totales de los consumidores del Ciclo de Pedido
```

Comment:
this is not a valid change. This will be overwritten by transifex. This change would have to be done in transifex.

Reply:
this should be removed.

Reply:
I think it's a whitespace difference when I remove the `reset_password_instructions.subject`

Reply:
now that you have one commit you can just do
git reset HEAD^1 -- config/locales/es.yml 
git commit --amend